apm-agent-core/src/test/java/co/elastic/apm/agent/impl/payload/TransactionPayloadJsonSchemaTest.java

```diff
@@ -1,293 +0,0 @@
-import static org.mockito.Mockito.mock;
-
-class TransactionPayloadJsonSchemaTest {
-
-    private JsonSchema schema;
-    private ObjectMapper objectMapper;
-    private DslJsonSerializer serializer;
-
-    @BeforeEach
-    void setUp() {
-        schema = JsonSchemaFactory.getInstance().getSchema(getClass().getResourceAsStream("/schema/transactions/payload.json"));
-        objectMapper = new ObjectMapper();
-        serializer = new DslJsonSerializer(mock(StacktraceConfiguration.class), mock(ApmServerClient.class));
-    }
-
-    private TransactionPayload createPayloadWithRequiredValues() {
-        final TransactionPayload payload = createPayload();
-        final Transaction transaction = createTransactionWithRequiredValues();
-        payload.getTransactions().add(transaction);
-        Span span = new Span(MockTracer.create());
-        span.start(TraceContext.fromParent(), transaction, -1, false)
-            .withType("type")
-            .withSubtype("subtype")
-            .withAction("action")
-            .withName("name");
-        payload.getSpans().add(span);
-        return payload;
-    }
-
-    private Transaction createTransactionWithRequiredValues() {
-        Transaction t = new Transaction(MockTracer.create());
-        t.start(TraceContext.asRoot(), null, (long) 0, ConstantSampler.of(true), getClass().getClassLoader());
-        t.withType("type");
-        t.getContext().getRequest().withMethod("GET");
-        t.getContext().getRequest().getUrl().appendToFull("http://localhost:8080/foo/bar");
-        return t;
-    }
-
-    private TransactionPayload createPayloadWithAllValues() {
-        final Transaction transaction = new Transaction(MockTracer.create());
-        TransactionUtils.fillTransaction(transaction);
-        final TransactionPayload payload = createPayload();
-        payload.getTransactions().add(transaction);
-        payload.getSpans().addAll(TransactionUtils.getSpans(transaction));
-        return payload;
-    }
-
-    private TransactionPayload createPayload() {
-        return createPayload(SystemInfo.create());
-    }
-
-    private TransactionPayload createPayload(SystemInfo system) {
-        Service service = new Service().withAgent(new Agent("name", "version")).withName("name");
-        final ProcessInfo processInfo = new ProcessInfo("title");
-        processInfo.getArgv().add("test");
-        return new TransactionPayload(processInfo, service, system);
-    }
-
-    @Test
-    void testJsonSchemaDslJsonEmptyValues() throws IOException {
-        final TransactionPayload payload = createPayload();
-        payload.getTransactions().add(new Transaction(MockTracer.create()));
-        final String content = new DslJsonSerializer(mock(StacktraceConfiguration.class), mock(ApmServerClient.class)).toJsonString(payload);
-        System.out.println(content);
-        objectMapper.readTree(content);
-    }
-
-    @Test
-    void testSystemInfo() throws IOException {
```

Comment:
Please also create a replacement for this

Reply:
added